ui/components/app/wallet-overview/aggregated-percentage-overview.test.tsx

```diff
@@ -603,3 +644,194 @@ describe('AggregatedPercentageOverview', () => {
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders correctly with positive values', () => {
+      mockGetHistoricalMultichainAggregatedBalance.mockReturnValue({
+        PT1H: { balance: 0, percentChange: 0, amountChange: 0 },
+        P1D: { balance: 1000, percentChange: 5.12, amountChange: 51.23 },
+        P7D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P14D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P30D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P200D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P1Y: { balance: 0, percentChange: 0, amountChange: 0 },
+      });
+
+      const { container } = render(<AggregatedMultichainPercentageOverview />);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders correctly with negative values', () => {
+      mockGetHistoricalMultichainAggregatedBalance.mockReturnValue({
+        PT1H: { balance: 0, percentChange: 0, amountChange: 0 },
+        P1D: { balance: 1000, percentChange: -4.87, amountChange: -51.23 },
+        P7D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P14D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P30D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P200D: { balance: 0, percentChange: 0, amountChange: 0 },
+        P1Y: { balance: 0, percentChange: 0, amountChange: 0 },
+      });
+
+      const { container } = render(<AggregatedMultichainPercentageOverview />);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  it('should display zero percentage and amount when balance is zero', () => {
+    render(<AggregatedMultichainPercentageOverview />);
+    const percentageElement = screen.getByTestId(
+      'aggregated-percentage-change',
+    );
+    const numberElement = screen.getByTestId('aggregated-value-change');
+    expect(percentageElement).toHaveTextContent('(+0.00%)');
+    expect(numberElement).toHaveTextContent('+$0.00');
+  });
+
+  it('should display positive percentage and amount change', () => {
+    mockGetHistoricalMultichainAggregatedBalance.mockReturnValue({
+      PT1H: { balance: 0, percentChange: 0, amountChange: 0 },
+      P1D: { balance: 1000, percentChange: 5.12, amountChange: 51.23 },
+      P7D: { balance: 0, percentChange: 0, amountChange: 0 },
+      P14D: { balance: 0, percentChange: 0, amountChange: 0 },
+      P30D: { balance: 0, percentChange: 0, amountChange: 0 },
+      P200D: { balance: 0, percentChange: 0, amountChange: 0 },
+      P1Y: { balance: 0, percentChange: 0, amountChange: 0 },
+    });
+
+    render(<AggregatedMultichainPercentageOverview />);
+    const percentageElement = screen.getByTestId(
+      'aggregated-percentage-change',
+    );
+    const numberElement = screen.getByTestId('aggregated-value-change');
+    expect(percentageElement).toHaveTextContent('(+5.12%)');
+    expect(numberElement).toHaveTextContent('+$51.23');
```

Comment:
nit (ignore) - I like these tests more than the snapshots. They kinda do the same thing as the snapshots... can we remove the snapshots? 😛